### PR TITLE
[DOCS] Added description for camera2D missing members

### PR DESCRIPTION
**Pull request by henriiquecampos**
Also fixed typo in AudioEffectFilter and fixed some descriptions to better understanding.

### CODE REVIEW

File: doc/classes/Camera2D.xml [hunk 1]
```diff
@@ -371,13 +375,13 @@
 			The camera's offset, useful for looking around or camera shake animations.
 		</member>
 		<member name="rotating" type="bool" setter="set_rotating" getter="is_rotating">
-			If [code]true[/code] the camera can rotate during gameplay. Default value: [code]false[/code].
+			If [code]true[/code] the camera rotates with the target.
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: Nice change, but don't remove the default value.

File: doc/classes/Camera2D.xml [hunk 2]
```diff
@@ -371,13 +375,13 @@
...
 		</member>
 		<member name="smoothing_enabled" type="bool" setter="set_enable_follow_smoothing" getter="is_follow_smoothing_enabled">
-			If [code]true[/code] the camera's velocity changes gradually, preventing sudden starts and stops. Default value: [code]false[/code].
+			If [code]true[/code] the camera smoothly moves towards the target at [member smoothing_speed].
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: Please keep the default value. It's useful to know at a glance that smoothing is off by default.

File: doc/classes/Camera2D.xml [hunk 3]
```diff
@@ -371,13 +375,13 @@
...
 		</member>
 		<member name="smoothing_speed" type="float" setter="set_follow_smoothing" getter="get_follow_smoothing">
-			How fast the camera's velocity changes if smoothing is enabled.
+			Speed in pixels per second of the smoothing effect of the camera.
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: - Use possessive over "of the"

"of the camera's smoothing effect."

- add link to associated member:

See [member smoothing_enabled].

**Review by cbscribe — CHANGES_REQUESTED** [review]

**Comment by henriiquecampos** [comment]
Thanks guys, done the changes, but please don't make me squash'm I'm kinda traumatized with this :(

**Comment by akien-mga** [comment]
If you follow the docs, there's nothing to be traumatized about: http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

And those docs even tell you that instead of making a new commit, you could have amended the original one directly.

**Review by cbscribe — APPROVED** [review]

**Comment by akien-mga** [comment]
Merged manually as 1063ddfeb.

**Pull request closed without merging**